src/routes/LoadSafePage/steps/ReviewLoadStep.tsx

```diff
@@ -35,16 +36,21 @@ function ReviewLoadStep(): ReactElement {
   const safeAddress = formValues[FIELD_LOAD_SAFE_ADDRESS] || ''
   const threshold = formValues[FIELD_SAFE_THRESHOLD]
   const ownerList = formValues[FIELD_SAFE_OWNER_LIST]
+  const ownersWithENSName = formValues[FIELD_SAFE_OWNER_ENS_LIST]
+
+  console.log(formValues)
 
   const ownerListWithNames = ownerList.map((owner) => {
     const ownerFieldName = `owner-address-${owner.address}`
-    const ownerNameValue = formValues[ownerFieldName]
+    const ownerNameValue = formValues[ownerFieldName] || ownersWithENSName[owner.address]
```

Comment:
This seems to be the same logic as in `getOwnerName`?

Reply:
Good catch!